about/introduction.rst

```diff
@@ -17,6 +21,33 @@ This page gives a broad presentation of the engine and of the contents
 of this documentation, so that you know where to start if you are a beginner or
 where to look if you need info on a specific feature.
 
+Where to start
+--------------
+
+So you're looking to get into Godot Engine - great!
+See the **About the documentation** section below for more details on how the
+Godot documentation is structured, but if you're just wondering
+*Where should I start?*, here are some pointers, depending on your preferences:
+
+**I am a complete beginner to game development.** |br|
+Reading this documentation in order is a good way to learn!
+So read on, finish this page and head on to the first section:
+:ref:`Step by step <toc-learn-step_by_step>`.
+
+**I already have some experience or just want to jump right in!** |br|
```

Comment:
I'm not sure I would advise "experienced" developers to skip the first parts of the step by step which introduce key concepts of scenes and nodes.

Reply:
Fair point, I guess it depends. If you know enough languages that GDScript isn't that surprising and enough engines that the concepts aren't totally foreign it IMO *is* a good point to start. It is what I did at least. :)

We can either add a sentence along the lines of "If you find you have trouble following the tutorial or want a more solid understanding, look at the content you skipped", or we drop the whole point.

IMO the step-by-step *does* start very slowly (foor good reasons), but maybe advanced people can be assumed to just skip ahead on their own.

I don't have a opinion on this. Any preference? The only thing important to me is directly pointing out the video tutorials on the very first page.

Reply:
I agree. A newcomer to the engine, even an experienced one, should see the nodes/scenes stuff. Let them decide if they want to skim/skip.